src/less/components/payex/button.less

```
@@ -27,16 +30,30 @@
 	}
 
 	&.btn-secondary {
+		border: 1px solid var(--brand-secondary-light-3);
+
 		&:focus {
-			border: 2px solid @brand-primary;
+			border: 1px solid var(--brand-secondary-light-3);
 			box-shadow: none;
 		}
 
 		&:active {
-			border: 1px solid var(--brand-primary);
+			border: 3px solid @btn-active-border-color;
```

when working on a design-system my approach is to try to introduce as few new variables as possibles, since new variables then need to be taken care of for a long time. And also try tp keep both sides (swedbankpay & payex) as in sync as possible (not regarding the actual values of the colro variables for example, but that the variables should be the same (once again, not the value per se e.g. "#3c3c3c", here I'm talking about the variable, e.g. `--brand-primary`)

also:
We are (slowly) trying to migrate progressively from "LESS" to pure CSS.
So I'd encourage to move away from LESS, not create new LESS specific variables.

this to say:
- instead of `border: 3px solid @btn-active-border-color;` I'd recommend setting the value : `border: 3px solid var(--brand-secondary);`
(since I've seen the color value you're setting on your variable is actually "#3c3c3c", which is also the brand-secondary color for PayEx if I'm not mistaking

```suggestion
			border: 3px solid var(--brand-secondary);
```